### PR TITLE
sep dev update

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/irods-development-update-september-2021.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+    - Python3 support (control scripts and Python rule engine plugin)
+    - now uses a PID file
+
+- [Imaging Working Group](https://github.com/irods-contrib/irods_working_group_imaging)
+
+    - https://github.com/irods-contrib/irods_working_group_imaging
+    - initial effort will be an OMERO integration
+    - working meeting
+        - Thur, September 16, 10am ET
+        - https://renci.zoom.us/j/5807924391
```
Comment (trel) [inline content/posts/irods-development-update-september-2021.md:55]: no links to meetings on the dev update

File: content/posts/irods-development-update-september-2021.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+    - https://github.com/irods-contrib/irods_working_group_s3
+    - develop tools to present iRODS as S3-compatible storage to existing S3 clients
+    - initial effort will be a MinIO gateway and integration services
+    - working meeting
+        - Tues, October 1, 3pm ET
+        - https://renci.zoom.us/j/5807924391
+
+- [TRiRODS](https://irods.org/trirods)
+
+    - https://irods.org/trirods
```
Comment (trel) [inline content/posts/irods-development-update-september-2021.md:86]: duplicate link

File: content/posts/irods-development-update-september-2021.md [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+    - [Metalnx](https://github.com/irods-contrib/metalnx-web)
+        - reimplemented search to use GenQuery only
+            - metadata and properties tab have been consolidated
+        - updated to use iRODS terminology (collections and data objects)
+        - interfaces are more consistent
+        - working on compatibility with the indexing plugin schema
+        - awaiting CI
+
+    - [C++ REST API](https://github.com/irods/irods_client_rest_cpp)
+        - https://github.com/irods/irods_client_rest_cpp
```
Comment (trel) [inline content/posts/irods-development-update-september-2021.md:120]: duplicate link

File: content/posts/irods-development-update-september-2021.md [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+    - [Logical Quotas](https://github.com/irods/irods_rule_engine_plugin_logical_quotas)
+    - [Hard Links Rule Engine Plugin](https://github.com/irods/irods_rule_engine_plugin_hard_links)
+        - awaiting more use cases before release
+        - https://github.com/irods/irods_rule_engine_plugin_hard_links
+    - [AWS S3 Lambda](https://github.com/irods/irods_client_aws_lambda_s3)
+        - future release
+            - could brute-force confirm checksums to detect renames
+            - would persist metadata
+        - could use atomic database operations to increase batch size > 1
+        - https://github.com/irods/irods_client_aws_lambda_s3
```
Comment (trel) [inline content/posts/irods-development-update-september-2021.md:242]: duplicate link

File: content/posts/irods-development-update-september-2021.md [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+- [TRiRODS](https://irods.org/trirods)
+
+    - https://irods.org/trirods
+    - Wed, December 1, 2021, 4pm ET
+    - speakers TBD
+
+- New Development Work
+
+    - new docker-based testing environment
+        - https://github.com/irods/irods_testing_environment
```
Comment (trel) [inline content/posts/irods-development-update-september-2021.md:93]: linkify (with previous line?)

File: content/posts/irods-development-update-september-2021.md [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+        - companion to iRODS development environment
+            - https://github.com/irods/irods_development_environment
```
Comment (trel) [inline content/posts/irods-development-update-september-2021.md:95]: linkify (with previous line?)

File: content/posts/irods-development-update-september-2021.md [hunk 7]
```diff
@@ -0,0 +1,264 @@
...
+Status: published
+
+
+Hello all! We've reached the end of another month and with that comes exciting iRODS news.
+
+We're excited to report that the [C++ REST API](https://github.com/irods/irods_client_rest_cpp) and [Zone Management Tool (ZMT)](https://github.com/irods/irods_client_zone_management_tool) are approaching their first official release! We have a few more issues to take care of before release, so look for those within a few days/weeks.
+
+The [indexing plugin](https://github.com/irods/irods_capability_indexing) is almost ready to be released. Support for the NIEHS schema has been implemented and tested. It is now in the final stages of polish.
+
+Metalnx has received quite a bit of changes in the past month. Search has been reimplemented to use GenQuery. This change allows Metalnx to honor permissions and present the same information as iquest of the icommands. The AVU search and Property search have been combinded to provide an interface closer to that of iquest. We hope the community finds these changes helpful moving forward.
```
Comment (trel) [inline content/posts/irods-development-update-september-2021.md:14]: bit -> number ?

Comment (trel) [reply content/posts/irods-development-update-september-2021.md:14]: combinded -> combined

Comment (trel) [reply content/posts/irods-development-update-september-2021.md:14]: double iquest mention

File: content/posts/irods-development-update-september-2021.md (hunk 1 above)
Comment (korydraughn) [reply content/posts/irods-development-update-september-2021.md]: done

File: content/posts/irods-development-update-september-2021.md (hunk 2 above)
Comment (korydraughn) [reply content/posts/irods-development-update-september-2021.md]: done

File: content/posts/irods-development-update-september-2021.md (hunk 3 above)
Comment (korydraughn) [reply content/posts/irods-development-update-september-2021.md]: done

File: content/posts/irods-development-update-september-2021.md (hunk 4 above)
Comment (korydraughn) [reply content/posts/irods-development-update-september-2021.md]: done

File: content/posts/irods-development-update-september-2021.md (hunk 5 above)
Comment (korydraughn) [reply content/posts/irods-development-update-september-2021.md]: done

**Comment by korydraughn** [comment]
I forgot to mention the changes for the globus plugin. Will those in a few.

**Comment by trel** [comment]
changes look good....   gotcha - will merge when you're ready.

File: content/posts/irods-development-update-september-2021.md [hunk 8]
```diff
@@ -0,0 +1,251 @@
...
+
+The [indexing plugin](https://github.com/irods/irods_capability_indexing) is almost ready to be released. Support for the NIEHS schema has been implemented and tested. It is now in the final stages of polish.
+
+Metalnx has received quite a number of changes in the past month. Search has been reimplemented to use GenQuery. This change allows Metalnx to honor permissions and present only the logical space. The AVU search and Property search have been combined to provide an interface closer to that of iquest. We hope the community finds these changes helpful moving forward.
+
+Jargon is getting support for Replica Access Tokens which enable parallel transfer over multiple streams. Development is going well and we hope to see a release soon. Big thanks to Mike Conway for his work.
+
+Like Jargon, we're working to add support for Replica Access Tokens to the [Globus Connector](https://github.com/irods/irods_client_globus_connector).
```
Comment (korydraughn) [inline content/posts/irods-development-update-september-2021.md:18]: @trel Thoughts on this line?

Comment (trel) [reply content/posts/irods-development-update-september-2021.md:18]: Seems good - anyone interested can click and read more.

**Comment by trel** [comment]
Let me know and I’ll merge.

File: content/posts/irods-development-update-september-2021.md (hunk 8 above)
Comment (korydraughn) [reply content/posts/irods-development-update-september-2021.md:18]: Cool. I think that's it.

**Comment by korydraughn** [comment]
I think it's ready to be merged.

**Pull request merged**